### PR TITLE
Fix manageiq_user tests

### PR DESCRIPTION
**Pull request by dkorn**

### CODE REVIEW

**Comment by dkorn** [comment]
@cben @zgalor @enoodle please review

**Review by zgalor — APPROVED** [review]
:+1: LGTM

File: tests/test_manageiq_user.py [hunk 1]
```diff
@@ -106,55 +109,54 @@ def fail(msg):
...
+        'msg': "Successfully created the user testuser: {}".format(POST_RETURN_VALUES['created_user']['results'])
     }
     miq.client.post.assert_called_once_with(
         '{hostname}/api/users'.format(hostname=MANAGEIQ_HOSTNAME),
         action='create',
         resource={'userid': USERID, 'name': USERNAME, 'password': PASSWORD, 'group': {'id': GROUP_ID}, 'email': EMAIL}
     )
 
-def update_user_email_and_name(miq, miq_api_class, the_user, the_group):
+def test_update_user_email_and_name(miq, miq_api_class, the_user, the_group):
```
Comment (cben) [inline tests/test_manageiq_user.py:129]: so previously these didn't run (without `test_` prefix), and all other changes are to actually make it work?

Comment (dkorn) [reply tests/test_manageiq_user.py:129]: exactly

**Review by cben — APPROVED** [review]

**Pull request merged**